django/contrib/gis/geos/prototypes/io.py

```diff
@@ -114,17 +114,10 @@ def __init__(self):
         self.ptr = self._constructor()
         # Loading the real destructor function at this point as doing it in
         # __del__ is too late (import error).
-        self._destructor.func = self._destructor.get_func(
-            *self._destructor.args, **self._destructor.kwargs
+        self.destructor.func = self.destructor.get_func(
```

Comment:
Once 312fc1af7b8a028611b45ee272652d59ea7bd85a is committed I think this could be just removed, correct?
I used `destructor` without underscore to be consistent with `ptr_type` from base class.